examples/index.html

```diff
@@ -6,12 +6,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"/>
     <meta http-equiv="x-ua-compatible" content="ie=edge"/>
 
-    <link rel="stylesheet" href="../node_modules/leaflet/dist/leaflet.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.9.2/leaflet.min.css">
```

Comment:
Here, i'm not against pegging a specific version via CDN, but was this necessary to make this work? Just curious if we could remove this to simplify the PR. Thanks!

Reply:
Hello ! Im alive !!!

Thx for your all yout comments !

The reason for moving to CDN was due to Github Pages service is failing when trying to load local files (missing files).

![image](https://user-images.githubusercontent.com/8063711/212836186-b5d7c5d0-98ac-4bbc-bbfd-329f0a27b0c2.png)

Then I couldn't test my changes on the web, lol, that's the only reason for this change.

Reply:
If needed I can set local method before the final merge ;)

Reply:
Oh that would be great to revert this back to the local one. We use that in our gh-pages demo. Thank you!